Process.h

```diff
@@ -26,6 +26,7 @@ in the source distribution for its full text.
 #include "Object.h"
 
 #define PROCESS_FLAG_IO 0x0001
+#define HIGHLIGHT_NSEC 5
```

Comment:
Shouldn't this be `HIGHLIGHT_SECS` as we are highlighting a bit™ longer than 5 nanoseconds?

Reply:
Haha, I use `nsec` as in `nbytes`. I'll change it.